plugins/GSdx/GSDevice11.cpp

```diff
@@ -768,7 +768,7 @@ void GSDevice11::InitExternalFX()
 			if (fshader.good())
 			{
 				shader << fshader.rdbuf();
-				CompileShader(shader.str().c_str(), shader.str().length(), shader_name.c_str(), "ps_main", NULL, &m_shaderfx.ps);
+				CompileShader(shader.str().c_str(), shader.str().length(), shader_name.c_str(), D3D_COMPILE_STANDARD_FILE_INCLUDE, "ps_main", nullptr, &m_shaderfx.ps);
```

Comment:
I see. 
But how would I be supposed to handle those special cases we all know?
Because I mean: if I had to use 8.1 SDK for example, then I'd need to mix headers to avoid stupid xinput 9.1.0 (and avoid xaudio2 2.8 in the future, but I didn't check it out)